src/gridstack.ts

```diff
@@ -327,7 +327,7 @@ export class GridStack {
       maxRow: this.opts.maxRow,
       onChange: (cbNodes) => {
         let maxH = 0;

```

Comment:
just set 
let maxH = this.opts.minRow || 0; 
instead so it doesn't check every time AND makes sure it not undefined

Reply:
code might be incorrect if undefined ? (or max(0, undefined) happens to work...